pkg/client/cache/store.go

```diff
@@ -116,6 +116,8 @@ type cache struct {
 	keyFunc KeyFunc
 }
 
+var _ Store = &cache{}
```

Comment:
I have an extremely minor preference for `var _ = Store(&cache{})`-- I think that's more common in our code base?